src/Microsoft.AspNetCore.ResponseCaching/ResponseCachingContext.cs

```diff
@@ -46,38 +46,88 @@ public bool CheckRequestAllowsCaching()
         }
 
         // Only QueryString is treated as case sensitive
-        // GET;HTTP://MYDOMAIN.COM:80/PATHBASE/PATH?QueryString
+        // GET;HTTP://MYDOMAIN.COM:80/PATHBASE/PATH?QueryString;VaryBy
```

Comment:
We'll need to finalize how keys are generated. Custom VaryBy, default components, etc.